.github/workflows/percy_snapshots.yml

```diff
@@ -26,6 +26,8 @@ on:
     secrets:
       PERCY_TOKEN:
         required: true
+      REACT_APP_MAPBOX_API_KEY:
```

Comment:
The test-site in the ui library was created with create-react-app so Yen set the env variable to pass through to `REACT_APP_MAPBOX_API_KEY`, as that's the only env variable format it can read I believe. We could make it just `MAPBOX_API_KEY` for use in storybook, but then you'd need to set both keys in the UI library `.env`. We should set that up once its needed outside of a react project

Reply:
we can still rename it to `MAPBOX_API_KEY` here for the reusable workflows. then in `search-ui-react`, you would just need to change the workflow to have it as `MAPBOX_API_KEY: ${{ secrets.REACT_APP_MAPBOX_API_KEY }}`

Reply:
I was thinking the same thing as Nidhi but I guess it would be a little weird since we would have to rename to have the REACT_APP prefix again below in the env part. I didn't know about the create react app env restrictions!

Reply:
[Here's](https://create-react-app.dev/docs/adding-custom-environment-variables/#:~:text=you%20change%20them.-,note%3A%20you%20must%20create%20custom%20environment%20variables%20beginning%20with%20react_app_.%20any%20other%20variables%20except%20node_env%20will%20be%20ignored,-to%20avoid%20accidentally) their reasoning. I can change to how nidhi described though